learn/advertising-web-3-0-new.ftd

```diff
@@ -0,0 +1,93 @@
+-- import: fifthtry.github.io/admint.io/assets
+-- import: fifthtry.github.io/admint.io/lib
+
+-- lib.page: Advertising &#038; Web 3.0 &#8211; Admint
+
+-- lib.tabs:
+
+-- lib.tab: Web 3.0 Basics
+link: learn/
+
+-- lib.tab: Brand Campaign
+link: learn/brand-campaigns/
+
+-- lib.tab: Advertising & Web 3.0
+active: true
+link: learn/advertising-web-3-0/
+
+-- end: lib.tabs
+
+-- lib.new-article: is the new playground for advertisment.
```

Comment:
@saurabh-lohiya what is `new-article`? just say `article` and change old `article` with `article-old`